static/styles/app_components.css

```diff
@@ -98,6 +98,10 @@ input::placeholder {
             border-radius: 4px;
         }
 
+        &.button-spacer {
```

Comment:
Instead of adding a new class, you can reference the button as like `.new-emoji-form #admin_emoji_submit`, also the full stop in commit message.

Reply:
> Instead of adding a new class, you can reference the button as like `.new-emoji-form #admin_emoji_submit`, also the full stop in commit message.

Done! :) 

Can you verify if I have done it correctly?